### DISCUSSION TITLE
Won't run on Windows 10 LTSC 2019 - claims .NET 4.8 is required

### DISCUSSION THREAD

**Discussion by Terri-Kennedy** [kangyu-california/PersistentWindows#230] [Q&A] [answered]
On Windows 10 LTSC 2019, installed from PersistentWindows5.41p1.zip, the program won't start. It says that .NET 4.8 is needed but not installed. I clicked on the Install button and downloaded ndp481-web.exe from Microsoft, but that won't install, saying "The .NET Framework 4.8.1 is not supported on this operating system." My system has "Microsoft .NET Runtime - 6.0.16 (x64)" installed.

I renamed PersistentWindows.exe.config to PersistentWindows.exe.noconfig to avoid this startup check, but I'm not sure if that is what is causing my next problem.

In any event, the download specifically for Win 10 Enterprise seems to have a prerequisite for something that isn't compatible with Win 10 Enterprise, which doesn't sound right.

**Comment by kangyu-california** [comment, marked as answer]
The .NET Framework 4.8 requirement is sort of typo, and can be fixed by editing PersistentWindows.exe.config as follows
sku=".NETFramework,Version=v4.5"

**Reply by Terri-Kennedy** [reply]
Thanks! That seems to have fixed it.
